src/main/webapp/javascript/bubbleDiagram/encounter-calls.js

```diff
@@ -45,14 +47,14 @@ var getData = function(individualID, displayName) {
 		
 				//var thisEncIndID = jsonData[i].encounters[j].individualID;   ///only when we fix thisOcc.encounters to be real json   :(
 				//console.info('i=%d, j=%d, -> %o', i, j, thisEncIndID);
-				if (!thisEncIndID || (thisEncIndID===displayName)) continue;  //unknown indiv -> false
-				if(encounterArray.includes(thisEncIndID)) {
-				} 
-				else {
-		    		encounterArray.push(thisEncIndID);
+				var individualName = thisEncIndID.split(" id=")[0];

```

Comment:
i think this might throw a js error if `thisEncIndID` is `false` ... so maybe we should have the `if (!thisEncIdId) continue;` block _before_ this.

Reply:
The check has been moved above now.